pyproject.toml

```diff
@@ -24,7 +26,7 @@ python = "^3.9"
 anthropic = "^0.23.1"
 backoff = "^2.2.1"
 bs4 = "^0.0.2"
-Flask = "^3.0.3"
+Flask = { version = "^3.0.3", optional = true }
```

Comment:
Make Flask optional to reduce conflicts with other repositories, as Flask is not required for the core OFV functionality.